test/parallel/test-global-descriptors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const assert = require('assert');
+
+const {
+  value,
+  configurable,
+  enumerable,
+  writable
+} = Object.getOwnPropertyDescriptor(global, 'global');
+
+assert.strictEqual(value, global, 'global should be global object');
```

Comment:
Good point, I'll change to `Function('return this')()` (i assume we're not worried about `Function` being overwritten in tests, or else i could use `function () {}.constructor('return this')()`)